.gitignore

```diff
@@ -12,3 +12,5 @@ cpu_codegen/
 
 .*.swp
 .nfs*
+build_cmake
```

Comment:
please remove the .gitignore

Reply:
the build_cmake is a reasonable gitignore i guess? we dont want to commit anything we generate in build_cmake

the test/**/dump.sh is generated from model tester. Again it never meant to be committed. So might as well add them to gitignore?